Stripe/STPPaymentContext.m

```diff
@@ -347,10 +497,51 @@ - (PKPaymentRequest *)buildPaymentRequest {
     NSArray<PKPaymentSummaryItem *> *summaryItems = self.paymentSummaryItems;
     paymentRequest.paymentSummaryItems = summaryItems;
     paymentRequest.requiredBillingAddressFields = [STPAddress applePayAddressFieldsFromBillingAddressFields:self.configuration.requiredBillingAddressFields];
+    paymentRequest.requiredShippingAddressFields = self.configuration.requiredShippingAddressFields;
     paymentRequest.currencyCode = self.paymentCurrency.uppercaseString;
+    if (self.selectedShippingMethod != nil) {
+        NSMutableArray<PKShippingMethod *>* orderedShippingMethods = [self.shippingMethods mutableCopy];
+        [orderedShippingMethods removeObject:self.selectedShippingMethod];
+        [orderedShippingMethods insertObject:self.selectedShippingMethod atIndex:0];
+        paymentRequest.shippingMethods = orderedShippingMethods;
+    }
+    else {
+        paymentRequest.shippingMethods = self.shippingMethods;
+    }
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_8_3
+    if ([paymentRequest respondsToSelector:@selector(shippingType)]) {
+        paymentRequest.shippingType = [[self class] pkShippingType:self.configuration.shippingType];;
+    }
+#endif
+    if (self.shippingAddress != nil) {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_9_0
+        if ([paymentRequest respondsToSelector:@selector(shippingContact)]) {
+            paymentRequest.shippingContact = [self.shippingAddress PKContactValue];
```

Comment:
I've added `STPAddress`->`PKContact` translation to work around a bug – the apple pay sheet doesn't appear on iOS10 if `shippingAddress` is set. I've filed a radar: https://openradar.appspot.com/radar?id=5518219632705536